src/xr/xr-input-source.js

```diff
@@ -224,6 +224,9 @@ Object.assign(pc, function () {
     XrInputSource.prototype._updateTransforms = function () {
         var dirty;
```

Comment:
Any reason to keep `dirty`?

Reply:
Yes, it uses it slightly bellow to decide to do transform update or not.